test/libsolidity/syntaxTests/storageLayoutSpecifier/constant_from_base_contract.sol

```diff
@@ -4,4 +4,4 @@ contract A {
 
 contract C is A layout at A.x { }
 // ----
-// TypeError 6396: (68-71): The base slot of the storage layout must evaluate to a rational number.
+// TypeError 1505: (68-71): The base slot expression cannot be evaluated during compilation.
```

Comment:
This one is weird. Why are we getting this and not something like:
```
Error: Member "x" not found or not visible after argument-dependent lookup in type(contract A).
```

Is `ConstantEvaluator` silencing this error and just returning `nullopt`?

Reply:
Yes, `ConstantEvaluator` can't handle member access. At the moment it can only process, literals, variables with no member acess from modules or other contracts and most of binary/unary operations.